helpers/cli_version_check/cli_version/installed_cli_version_test.go

```diff
@@ -0,0 +1,34 @@
+package cli_version_test
+
+import (
+	"github.com/cloudfoundry/cf-acceptance-tests/helpers/cli_version_check/cli_version"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NewInstalledCliVersion", func() {
+	var ver cli_version.InstalledCliVersion
+
+	BeforeEach(func() {
+		ver = cli_version.NewInstalledCliVersion()
+	})
+
+	Context("GetVersion", func() {
+		It("returns a version value that only contains numbers and dots", func() {
+			ver.SetFullVersionString("abcdds1.2.3dksfj")
+			Ω(ver.GetVersion()).To(Equal("1.2.3"))
+
+			ver.SetFullVersionString("5.2.3  dksfj")
+			Ω(ver.GetVersion()).To(Equal("5.2.3"))
+
+			ver.SetFullVersionString("____ 3.4.3 -dksfj")
+			Ω(ver.GetVersion()).To(Equal("3.4.3"))
```

Comment:
Realistically, what cases are these assertions trying to test?  When does the CLI version have all these extra characters?